internal/commands/base.go

```diff
@@ -89,6 +89,13 @@ var (
 func init() {
 	rootCmd.PersistentFlags().StringVarP(&ll, "log-level", "l", "", "enable debug message")
 	rootCmd.PersistentFlags().StringVarP(&cfgFile, "config-file", "c", "", "set config file name")
```

Comment:
Would this be available via `IZE_CONFIG_FILE`? if not, let's convert it too please